spec/graphql/schema/field_spec.rb

```diff
@@ -36,5 +36,43 @@
       assert_equal "test", object.fields["test"].arguments["test"].name
       assert_equal "A Description.", object.fields["test"].description
     end
+
+    describe "complexity" do
+      it "accepts a keyword argument" do
+        object = Class.new(Jazz::BaseObject) do
+          graphql_name "complexityKeyword"
+
+          field :complexityTest, String, null: true, complexity: 25
+        end.to_graphql
+
+        assert_equal 25, object.fields["complexityTest"].complexity
+      end
+
+      it "accepts a proc in the definition block" do
+        object = Class.new(Jazz::BaseObject) do
+          graphql_name "complexityKeyword"
+
+          field :complexityTest, String, null: true do
+            complexity ->(_ctx, _args, _child_complexity) { 52 }
+          end
+        end.to_graphql
+
+        assert_equal 52, object.fields["complexityTest"].complexity.call(nil, nil, nil)
+      end
+
+      it 'fails if the proc does not accept 3 parameters' do
+        err = assert_raises(RuntimeError) do
+          Class.new(Jazz::BaseObject) do
+            graphql_name "complexityKeyword"
+
+            field :complexityTest, String, null: true do
+              complexity ->(one, two) { 52 }
+            end
+          end.to_graphql
+        end
+
+        assert_equal true, !!err.message.match(/^A complexity proc should always accept 3 parameters/)
```

Comment:
for nicer error messages, you can also use [assert_match](http://devdocs.io/minitest/minitest/assertions#method-i-assert_match)